src/pages/asset/[did]/edit.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from 'react'
+import { useRouter } from 'next/router'
+import EditPage from '../../../components/Asset/Edit'
+import AssetProvider from '@context/Asset'
+
```

Comment:
this route needs to be protected (render alert instead of page), so it can only be reached by the owner of the asset. Not sure where best place is to do that, most likely within `EditPage` as there you have access to `asset` from AssetProvider